Microsoft.Quantum.Canon/Arrays/Arrays.qs

```diff
@@ -252,5 +253,52 @@ namespace Microsoft.Quantum.Canon {
 		return sliced;
 	}
 
+    /// # Summary
+    /// Returns an array padded at the tail with specified values up to a 
+    /// specified length.
+    ///
+    /// # Type Parameters
+    /// ## 'T
+    /// The type of the array elements.
+    ///
+    /// # Input
+    /// ## nElementsTotal
+    /// The length of the padded array. If this is positive, `inputArray`
```

Comment:
This convention leads to the counterintuitive behavior that `PadTail` can add things at the head; I'd suggest either splitting into two functions (`PadHead` and `PadTail`) or renaming the single function to `Pad` to avoid confusion.

Reply:
Function renamed to `Pad`. Function now pads at head when `nElementsTotal` is positive.